frontend/src/metabase/components/HeaderBar.jsx

```diff
@@ -23,12 +24,12 @@ export default class Header extends Component {
                 </div>
             );
         } else {
-            if (name && description) {
+            if(name && description) {
```

Comment:
Why do you remove space between if and ( ? I suppose coding standard need space..